pages/villagers.jsx

```diff
@@ -0,0 +1,35 @@
+import useFetch from '../useFetch';
+import { useState, useEffect } from 'react';
+import CardsWrapper from '../components/CardsWrapper';
+import VillagersCard from '../components/VillagerCard';
```

Comment:
Du könntest in erwägung ziehen, reine styled components in z.B. einen ui-component ordner zu legen. Damit man unterscheiden kann, welche Komponenten Logic enthalten